### DISCUSSION TITLE
2007/04/18/%E5%91%98%E5%B7%A5%E4%BB%AC%E7%9A%84%E5%9B%9A%E5%BE%92%E5%9B%B0%E5%A2%83

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#153] [Announcements]
# 2007/04/18/%E5%91%98%E5%B7%A5%E4%BB%AC%E7%9A%84%E5%9B%9A%E5%BE%92%E5%9B%B0%E5%A2%83

几个月前看了一本 James Miller 写的《Game Theory at Work》，一直想写读后感的，可是一直没肯花时间。现在总算有心情写了。

https://blog.qizhen.xyz/2007/04/18/%E5%91%98%E5%B7%A5%E4%BB%AC%E7%9A%84%E5%9B%9A%E5%BE%92%E5%9B%B0%E5%A2%83

**Comment by ruanqizhen** [comment]
之前博客下的留言：

xia
2007年04月20日 20:29 编辑
感觉又像在上微观经济学课了~汗~
——————————
路过~

回复
Qizhen
2007年04月20日 19:36 编辑
关于上msn，一是因为我现在在美国，跟大家上网的时间都错开了。二是由于公司的限制，除了工作上的联系人，现在已经不填加别人到我的msn了。所以如果有问题问我，还是留言给我吧。

回复
晨恺
2007年04月20日 14:01 编辑
你从来都不上MSN的?

回复
Qizhen
2007年04月20日 07:54 编辑
Game 有很多意思，它在 Game Theory 中，本意是指下棋。博弈这个词愿意就是下棋。因此 Game Theory 被翻译成了博弈论。

回复
杰
2007年04月19日 21:16 编辑
博弈，这个翻译的很妙。呵呵。game theory.老外意思就是游戏的理论，行为策略的理论。
不过确实是有点博大精深。看过一点原版的博弈论，很累。
它对我们生活中的人与人之间的行为做出理论指导，可惜我们总是一意孤行，还对自己说，这叫个性。呵呵。
记得纳什均衡是一个理想的抽象掉时间的一个概念，我们都在人群中，如果我的任何一种行为都不能让自己获得更大的利益，那么我会保持原来的行为，再推一步，对任何一个人来说，都会保持当前的行为，那么就认为，此刻纳什均衡。就好像我每天都要上班一样。不过实际情况来说，都是多次博弈的结果，譬如说有个猎头找我给我更好的工作环境和更高的薪水…我觉得这道理只是告诉我，努力提升自己的价值，去打破这见鬼的均衡。呵呵。
PS：我未婚妻学经济学的，家里好多原版的大部头。经济学，感觉应该和数学系合并。当然，只是某种意义上。

回复
Dchneric
2007年04月19日 18:35 编辑
囚徒困境恰好是（非）零和对策有鞍点的情况亚。在盈利矩阵里一方的最优解恰好也是另一方的，呵呵 在一些应用数学书里有讲，是满好玩的。如果没有鞍点就要算概率了。

回复
yoyo
2007年04月18日 22:08 编辑
只看了一点点，光记得有一条是赌钱稳赢的招数，看来得把书找出来好好看

回复
Qizhen
2007年04月18日 20:04 编辑
书非借不能读也，我跟你最大的区别就是：书是你的，我是借的:D

回复
永青
2007年04月18日 13:46 编辑
看来我有必要重读此书啊！
我咋就没觉得有那么大收获呢？
